src/Glimpse.Server.Web/Resources/History.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Glimpse.Server.Web.Extensions;
+using Microsoft.AspNet.Http;
+using Microsoft.Net.Http.Headers;
+
+namespace Glimpse.Server.Web
+{
+    public class History : IResource
```

Comment:
Not 100% passionate either way. More pointing to it being different to the others and it should either be the same for this PR and then, if we want to update them a different task to update all of them. 

I personally like having Resource as a post fix, it feels like Controllers in MVC with `HomeController`. That said, it doesn't mean we have to follow that lead if its going to give us issues.